src/components/pages/manage/productions/create/index.js

```diff
@@ -50,19 +50,31 @@ const Create = () => {
     await refetchSuggestedEvents();
   };
 
-  const { mutate: skipSuggestedEvents } = useSkipSuggestedEvents({
+  const {
+    mutate: skipSuggestedEvents,
+    ...skipSuggestedEventsMutation
+  } = useSkipSuggestedEvents({
```

Comment:
Can't we just use skipSuggestedEventsMutation.mutate everywhere? The mix of restructuring and renaming is a bit confusing imo.

Reply:
Changed it